src/pages/EventDetails/EventDetails.js

```diff
@@ -1,15 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Header from '../../components/Header/Header';
 
-const EventDetails = () => {
+const propTypes = {
+  match: PropTypes.object.isRequired
+};
+
+const EventDetails = (props) => {
   return (
       <div className='content-container'>
-        <Header
-          title={'Breminale-Web'}
-        />
-      <img src={"https://via.placeholder.com/350x150"} />
+        <Header title={'Breminale-Web'} />
+        <div className='event-container'>
+          <div className='event-banner'>
+            <img src={'http://www.joko-bremen.de/breminale15/SAM_1657.jpg'} />
+          </div>
+          <div className='event-description'>
+            <p>This is the Event with ID {props.match.params.id}</p>
```

Comment:
params might be null? If an empty object is passed right?

Reply:
params can't be null, it's available as soon as the component gets mounted. And this will always happen when someone enters the page.